streams/src/test/java/org/apache/kafka/streams/kstream/internals/KStreamTransformValuesTest.java

```diff
@@ -54,92 +44,6 @@ public class KStreamTransformValuesTest {
     private final Properties props = StreamsTestUtils.getStreamsConfig(Serdes.Integer(), Serdes.Integer());
     private InternalProcessorContext context = mock(InternalProcessorContext.class);
 
-    @SuppressWarnings("deprecation") // Old PAPI. Needs to be migrated.
-    @Test
-    public void testTransform() {
-        final StreamsBuilder builder = new StreamsBuilder();
-
-        final org.apache.kafka.streams.kstream.ValueTransformerSupplier<Number, Integer> valueTransformerSupplier =
-            () -> new org.apache.kafka.streams.kstream.ValueTransformer<Number, Integer>() {
-                private int total = 0;
-
-                @Override
-                public void init(final org.apache.kafka.streams.processor.ProcessorContext context) { }
-
-                @Override
-                public Integer transform(final Number value) {
-                    total += value.intValue();
-                    return total;
-                }
-
-                @Override
-                public void close() { }
-            };
-
-        final int[] expectedKeys = {1, 10, 100, 1000};
-
-        final KStream<Integer, Integer> stream;
-        stream = builder.stream(topicName, Consumed.with(Serdes.Integer(), Serdes.Integer()));
-        stream.transformValues(valueTransformerSupplier).process(supplier);
-
-        try (final TopologyTestDriver driver = new TopologyTestDriver(builder.build(), props)) {
-            for (final int expectedKey : expectedKeys) {
-                final TestInputTopic<Integer, Integer> inputTopic =
-                        driver.createInputTopic(topicName, new IntegerSerializer(), new IntegerSerializer());
-                inputTopic.pipeInput(expectedKey, expectedKey * 10, expectedKey / 2L);
-            }
-        }
-        final KeyValueTimestamp[] expected = {new KeyValueTimestamp<>(1, 10, 0),
-            new KeyValueTimestamp<>(10, 110, 5),
-            new KeyValueTimestamp<>(100, 1110, 50),
-            new KeyValueTimestamp<>(1000, 11110, 500)};
-
-        assertArrayEquals(expected, supplier.theCapturedProcessor().processed().toArray());
-    }
-
-    @SuppressWarnings("deprecation") // Old PAPI. Needs to be migrated.
```

Comment:
as above

Reply:
I think they both are covered by `org.apache.kafka.streams.kstream.internals.KStreamImplTest#shouldProcessValues`, since `processValues` just has `FixedKeyProcessorSupplier` as processor supplier.

Reply:
Thanks. SG.

But I am wondering if we can remove `KStreamTransformValuesTest.java` completely for this case?

What also implies that we can remove `KStreamTransformValues.java`?

Reply:
@fonsdant Did you miss this comment?

Reply:
@mjsax, oh, sorry! Yes, I have missed it. Thanks!

Makes sense to me too! I'm going to push the removal.

Reply:
> Following this approach, we could remove KStreamFlatTransformTest and KStreamFlatTransform too, couldn't we?

Maybe in PR [4/4] but not in this PR, right?

Reply:
> Following this approach, we could remove KStreamFlatTransformTest and KStreamFlatTransform too, couldn't we?

Maybe in PR [4/4], but not in this PR, right?